cmd/git-chglog/main.go

```diff
@@ -76,6 +80,12 @@ func CreateApp(actionFunc cli.ActionFunc) *cli.App {
 			Usage: "generate the git-chglog configuration file in interactive",
 		},
 
+		// path
+		cli.StringSliceFlag{
```

Comment:
```suggestion
		&cli.StringSliceFlag{
```

Reply:
Hopefully new commits resolves this concern.

Reply:
Thanks for pushing the updates. Looks good!